change/@fluentui-react-3a2d2c1d-90af-4cef-971b-d676ce98dab3.json

```diff
@@ -1,7 +0,0 @@
-{
-  "type": "prerelease",
```

Comment:
Was this intentionally deleted?

Reply:
oops, nope, definitely not intentional. I'll go see what happened with these 😅